bin/change_axisorder.py

```diff
@@ -1,35 +1,102 @@
+#!/usr/bin/env python3
+"""
+HDF5 Axis Order Modifier for ILASTIK Projects
+
+This script modifies the axis order in ILASTIK Project (.ilp) files.
+ILASTIK is an interactive learning and segmentation toolkit, and its
+project files use HDF5 format to store configuration and data.
+
+The script changes the axis order of a dataset and removes any existing
+classifier to force retraining with the new axis configuration.
+
+Usage:
+    python modify_axis_order.py project_file.ilp new_axisorder
+
+Example:
+    python modify_axis_order.py segmentation.ilp "txyzc"
+"""
+
 from __future__ import print_function
 import sys
 import h5py
 
+
+def modify_axis_order(projectfile, axisorder):
+    """
+    Modify the axis order in an ILASTIK project file.
+    
+    Args:
+        projectfile (str): Path to the ILASTIK project file (.ilp)
+        axisorder (str): New axis order string (e.g., "txyzc")
+        
+    Returns:
+        bool: True if modification was successful
+    """
+    try:
```

Comment:
Checking the arguments in __main__ and moving out the implementation into a function is nice, but I think this try-except wrap for a success return code adds unnecessary complexity. If there are potential errors that one can anticipate here, the code should communicate what they are through a more specific `except`, and otherwise if no specific errors are expected, then unexpected errors should just bubble up and terminate the script with a more informative stack trace.